ElectronClient/gui/ConfigScreen.jsx

```diff
@@ -292,6 +295,7 @@ class ConfigScreenComponent extends React.Component {
 			}
 
 			updateSettingValue('theme', settings[changeSetting]);
+			Setting.setValue('theme', settings[changeSetting]);
```

Comment:
Using this line active theme will be instantly set to the preferred light/dark theme based on detected system theme so that user can know which theme is detected by Auto-Detect option.